activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ConnectorValidationControllerIT.java

```diff
@@ -252,14 +252,50 @@ public void should_returnStatusNoContent_when_validatingConnectorWithCustomTypes
         final Model connectorModel = modelRepository.createModel(connectorModel("connector-name"));
 
         given()
-            .multiPart("file",
-                "connector-with-custom-type.json",
-                resourceAsByteArray("connector/connector-with-custom-type.json"),
-                "text/plain")
-            .post("/v1/models/{modelId}/validate",
-                connectorModel.getId())
-            .then()
-            .expect(status().isNoContent())
-            .body(isEmptyString());
+                .multiPart("file",
+                        "connector-with-custom-type.json",
+                        resourceAsByteArray("connector/connector-with-custom-type.json"),
+                        "text/plain")
+                .post("/v1/models/{modelId}/validate",
+                        connectorModel.getId())
+                .then()
+                .expect(status().isNoContent())
+                .body(isEmptyString());
+    }
+
+    @Test
+    public void should_returnStatusNoContent_when_validatingConnectorWithErrors() throws IOException {
+        final Model connectorModel = modelRepository.createModel(connectorModel("connector-name"));
+
+        given()
+                .multiPart("file",
+                        "connector-with-events.json",
+                        resourceAsByteArray("connector/connector-with-events.json"),
```

Comment:
probably `connector-with-errors.json` was intended? Otherwise that .json resource is unused

Reply:
Well spotted. Thank you @zoltanpalfi.